public/app/AppWrapper.tsx

```diff
@@ -144,3 +131,93 @@ export class AppWrapper extends Component<AppWrapperProps, AppWrapperState> {
     );
   }
 }
+
+function RouterTree(props: { routes?: JSX.Element | false }) {
+  return (
+    <Router history={locationService.getHistory()}>
+      <LocationServiceProvider service={locationService}>
+        <CompatRouter>
+          <ModalsContextProvider>
+            <GlobalStyles />
+            <AppChrome>
+              <AngularRoot />
+              <AppNotificationList />
+              <Stack gap={0} grow={1} direction="column">
+                {pageBanners.map((Banner, index) => (
+                  <Banner key={index.toString()} />
+                ))}
+                {props.routes}
+              </Stack>
+              {bodyRenderHooks.map((Hook, index) => (
+                <Hook key={index.toString()} />
+              ))}
+            </AppChrome>
+            <LiveConnectionWarning />
+            <ModalRoot />
+            <PortalContainer />
+          </ModalsContextProvider>
+        </CompatRouter>
+      </LocationServiceProvider>
+    </Router>
+  );
+}
+
+/**
+ * Renders both the main app tree and a secondary sidecar app tree to show 2 apps at the same time in a resizable split
+ * view.
+ * @param props
+ * @constructor
+ */
+function ExperimentalSplitPaneTree(props: { routes?: JSX.Element | false }) {
+  const appId = useSelector((state) => state.appSidecar.appId);
+  const dispatch = useDispatch();
+
+  const memoryLocationService = new HistoryWrapper(H.createMemoryHistory({ initialEntries: ['/'] }));
+
+  return (
+    <SplitPaneWrapper
+      splitOrientation="vertical"
+      paneSize={0.25}
+      minSize={200}
+      maxSize={200 * -1}
+      primary="second"
+      splitVisible={!!appId}
+      paneStyle={{ overflow: 'auto', display: 'flex', flexDirection: 'column' }}
```

Comment:
can you try using the useSplitter hook instead?

Reply:
Yes changed it here https://github.com/grafana/grafana/pull/91648/commits/a6086abaec1f76646b5495846cbb0e6f718c41b7, didn't know we had such a thing